### DISCUSSION TITLE
homing issue and priting gradually offsets

### DISCUSSION THREAD

**Discussion by dragoonjewels** [arkypita/LaserGRBL#1454] [Q&A]
I am using the twotrees totem laser engraver and the lasergrbl v4.3.0 I am new to this and the first question is i am getting homing failed error when trying to home and the second is that my images start out fine but as the burning persists it gradually offsets attached are the screen shots of my settings
![settings 1](https://user-images.githubusercontent.com/83870633/117540643-1ed7ed00-afde-11eb-9398-568ad635ae0e.jpg)
![settings 2](https://user-images.githubusercontent.com/83870633/117540653-24353780-afde-11eb-9f16-6bde69191fd9.jpg)

Here is the burn from the image i tried

![Screenshot 2021-05-08 092415](https://user-images.githubusercontent.com/83870633/117540863-3368b500-afdf-11eb-9f81-1cdc4e60d88e.jpg)

**Comment by gmmanonymus111** [comment]
AFAIK TwoTrees engraver does not have end stops, so you just can't home your machine. (Homing will fail, because it waits for the end switch to be pushed. Since your engraver does not have any, GRBL times out. In the process, it may crash your X/Y/Z axis to the end of your machine, and may cause permanent damage.)
This offset/line shifting might be caused because of your too high acceleration (Try setting `$120` & `$121` to ~50)

**Reply by dragoonjewels** [reply]
this is now what i get 
![after changing to 50](https://user-images.githubusercontent.com/83870633/117543048-2a7ce100-afe9-11eb-9eef-fe758638f9ab.jpg)

**Reply by dragoonjewels** [reply]
i think i have it 500 instead of 50

**Reply by dragoonjewels** [reply]
omg you are a mirical worker can i have your personal contact I love you.... LOL

**Reply by dragoonjewels** [reply]
now another question i am trying to burn on wood and the smax is set to 1000 and it is not burning how can i fix that

**Reply by dragoonjewels** [reply]
nevermind i figured it out i had to slow the speed down

**Reply by dragoonjewels** [reply]
ok i didnt fix it how do i change the density of the lazer to burn on wood

**Reply by gmmanonymus111** [reply]
Did you focus your laser properly? It has to be as small of a dot (rectangle) as possible. You can also use LaserGRBL's material DB to automatically set the correct speed/laser power for a specific material
